### ISSUE TITLE
MinQuantity of 0 resetting to 1

### ISSUE THREAD

**Issue by chdavi** [thommcgrath/Beacon#67]
The valid lower range for MinQuantity in ARK is 0. This is used for to limit blueprints in the original beacon data, and I typically use it for the same thing. Example: Full set of armor + a chance of a blueprint.

However, when I try and set the MinQuantity of an item in a set to 0 in Beacon, it resets it to 1.

**Comment by thommcgrath** [comment]
So for a real world example, let’s say you have an item set with a guaranteed 6 items: head, chest, arms, legs, boots, and blueprint of any of those. The blueprint would have a quantity min of 0, so that the 5 pieces are guaranteed, but the blueprint is only a possibility. Does that sound right?

**Comment by chdavi** [comment]
Exactly. I use it a lot with armor sets, housing sets and weapon sets.

**Comment by thommcgrath** [comment]
Ok, I think that makes sense. I’ll get it done, no ETA though.

**Comment by chdavi** [comment]
Thanks!
